tests/Utilities.hs

```diff
@@ -321,9 +330,15 @@ iscanr f z = scanr (uncurry f) z . zip [0..]
 ifoldr :: (Int -> a -> b -> b) -> b -> [a] -> b
 ifoldr f z = foldr (uncurry f) z . zip [0..]
 
-ifoldM :: Monad m => (a -> Int -> a -> m a) -> a -> [a] -> m a
+ifoldM :: Monad m => (b -> Int -> a -> m b) -> b -> [a] -> m b
 ifoldM = indexedLeftFold foldM
 
+foldrM :: Monad m => (a -> b -> m b) -> b -> [a] -> m b
+foldrM f z = foldr (\a mb -> f a =<< mb) (pure z)
```

Comment:
How come we need to define it instead of using `Data.Foldable.foldrM` instead?

Reply:
I forgot about it :)